Old packages won't install ('invalid filename' error)

Using master of sinopia:

Client:

```
$ npm install thirty-two --registry=http://localhost:4873/
npm http GET http://localhost:4873/thirty-two
npm http 200 http://localhost:4873/thirty-two
npm http GET http://localhost:4873/thirty-two/-/thirty-two@0.0.1.tgz
npm http 403 http://localhost:4873/thirty-two/-/thirty-two@0.0.1.tgz
npm ERR! fetch failed http://localhost:4873/thirty-two/-/thirty-two@0.0.1.tgz
npm ERR! Error: 403 Forbidden
...
npm ERR! npm -v 1.4.3
```

Server:

```
$ bin/sinopia 
 warn  --- Server is listening on http://localhost:4873/
 http  <-- 304, user: undefined, req: 'GET /thirty-two', bytes: 0/0
 http  <-- 403, user: undefined, req: 'GET /thirty-two/-/thirty-two@0.0.1.tgz', error: invalid filename
```

--- a/lib/utils.js
+++ b/lib/utils.js
@@ -10,7 +10,7 @@ module.exports.validate_name = function(name) {
 	if (
 		name.charAt(0) === '.' || // ".bin", etc.
 		name.charAt(0) === '-' || // "-" is reserved by couchdb
-		name.match(/[\/\s\+%:]/) ||
+		name.match(/[\\\/\s\+%:]/) ||
 		name === 'node_modules' ||
 		name === '__proto__' ||
 		name === 'package.json' ||

--- a/test/unit/utils.js
+++ b/test/unit/utils.js
@@ -5,6 +5,7 @@ describe('Validate', function() {
 	it('good ones', function() {
 		assert(validate('sinopia'))
 		assert(validate('some.weird.package-zzz'))
+		assert(validate('old-package@0.1.2.tgz'))
 	})
 
 	it('uppercase', function() {
@@ -32,7 +33,6 @@ describe('Validate', function() {
 	})
 
 	it('other', function() {
-		assert(!validate('pkg@'))
 		assert(!validate('pk g'))
 		assert(!validate('pk\tg'))
 		assert(!validate('pk%20g'))

